src/osemgrep/targeting/Glob_parser.mly

```diff
@@ -0,0 +1,37 @@
+%{
+open Glob_matcher
+
+type tmp =
+| Fragment of component_fragment
+| Starstar
+
+let convert_ellipses (fragments : tmp list) =
+  match fragments with
+  | [Starstar] -> Ellipsis
+  | xs ->
+     Component (Common.map (function Fragment x -> x | Starstar -> Star) xs)
+%}
+%token SLASH QUESTION STAR STARSTAR EOF
+%token <char> CHAR
+%token <Glob_matcher.char_class> CHAR_CLASS
+
+%start <Glob_matcher.component list> components
+%%
+
+components:
+| frags=list(fragment) SLASH comps=components
```

Comment:
not a fan of those xxx=yyy. I prefer $1 $2, but whatever.

Reply:
We should get you this 😂
![image](https://user-images.githubusercontent.com/343265/222099389-88a4b68d-bbb7-4db0-9e51-83a838b4e323.png)

Reply:
We should get you this 😂
![image](https://user-images.githubusercontent.com/343265/222099906-165d3bbc-8235-4bf4-8615-40bb078b1c45.png)